### PR TITLE
Gracefully handle keyboard interupt (SIGINT)

### PR DESCRIPTION
**Pull request by vfrazao-ns1**
Relates to issue: https://github.com/dalibo/pg_activity/issues/209

First time contributor - please let me know if I'm missing anything for this PR (I didn't see any contributing or dev guideline).

### CODE REVIEW

File: pgactivity/__main__.py [hunk 1]
```diff
@@ -1,4 +1,10 @@
...
+
 from .cli import main
 
 if __name__ == "__main__":
-    main()
+    try:
+        main()
+    except KeyboardInterrupt:
+        sys.exit(1)
+
```
Comment (dlax) [inline pgactivity/__main__.py:10]: This change does seem needed, as the change in `cli.main()`, below is sufficient I think.

(This file is used as `python3 -m pgactivity [ARGS]`.)

**Review by dlax — CHANGES_REQUESTED** [review]
Thank you for your contribution.

It looks good to me overall, just one thing apparently superfluous.

File: pgactivity/__main__.py (hunk 1 above)
Comment (vfrazao-ns1) [reply pgactivity/__main__.py]: Yes, thank you for pointing that out. I've changed `__main__.py` back to match master.

**Review by dlax — APPROVED** [review]
LGTM. Will merge when CI completes.

Thanks!

**Pull request merged**